trio/_signals.py

```diff
@@ -139,14 +139,14 @@ def open_signal_receiver(*signals):
 
     Example:
 
-      A common convention for Unix daemons is that they should reload their
-      configuration when they receive a ``SIGHUP``. Here's a sketch of what
-      that might look like using :func:`open_signal_receiver`::
+      In order to terminate a program the ``SIGTERM`` signal, which is also supported on 
+      windows, can be send. Here's a sketch of what that might look like using
```

Comment:
```suggestion
      Windows. Here's a sketch of what that might look like using
```

Reply:
I think this was a typo and should read `can be sent`, that makes sense if you read the rest of it in context.